src/mod_ping.erl

```diff
@@ -56,7 +56,7 @@ start_ping_timer(HandlerState, Server) ->
     ejabberd_c2s_info_handler:call_after(PingInterval * 1000, self(), mod_ping, send_ping).
 
 route_ping_iq(JID, Server) ->
-    PingReqTimeout = gen_mod:get_module_opt(Server, ?MODULE, ping_req_timeout,
+    PingReqTimeout = 1000 * gen_mod:get_module_opt(Server, ?MODULE, ping_req_timeout,
```

Comment:
I'd rather go for `timer:seconds(...)`, it's more readable than multiplying by a thousand. Same could be done above, in the call to `ejabberd_c2s_info_handler:call_after/4`